src/Components/sidenav/Sidenav.jsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router-dom";
+import "./sidenav.css"
+import "../../styles.css"
+export function Sidenav() {
+  return (
+    <div className="sidenav py-4 px-2">
+      <ul className="list-group-stacked">
+        <Link to="/" className="link"><li className="list-item txt-center">
```

Comment:
`li` should be the direct child of `ul` nest `Link` inside `li`

Reply:
Done!